packages/l/libkmod/port/elf.h

```diff
@@ -0,0 +1,3558 @@
+/* This file defines standard ELF types, structures, and macros.
+   Copyright (C) 1995-2015 Free Software Foundation, Inc.
```

Comment:
it's too large. please use `libelf` or `elfutils` package

Reply:
removed